app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class Inventory < ManagerRefresh::Inventory::Core::Inventory
+    def credentials
```

Comment:
It's up there in the [persister](https://github.com/ManageIQ/manageiq/pull/14022/files#diff-f5679d6859cb484bf3beef23dbce89e1R4)

```
has_inventory automation_manager.credentials, :association => :something_else
```

is that what you mean?